### DISCUSSION TITLE
multiple motor's

### DISCUSSION THREAD

**Discussion by hydrometeo201** [mwood77/winderoo#48] [Ideas]
Hi
I want to convert my 6 watch's winder to esp32 wifi.
Is it possible?

Thanks

**Comment by mwood77** [comment]
Should be! 

You’ll have to figure out the wiring yourself, but it should be pretty straightforward. However, Winderoo will not be able to set individual turns per day, or winding direction per watch; it’s all or nothing.

**Reply by hydrometeo201** [reply]
ok thanh you very much
